packages/pixeloven-webpack-config/src/configs/webpack/client.test.ts

```diff
@@ -0,0 +1,41 @@
+import * as core from "@pixeloven/core";
+import "jest";
+import sinon, { SinonSandbox } from "sinon";
+import client from "./client";
+
+let sandbox: SinonSandbox;
+
+describe("@pixeloven/webpack", () => {
+    describe("configs", () => {
```

Comment:
If you wanted to reduce the nesting the final output is just a space between each so you could write one describe statement as `@pixeloven/webpack configs webpack`. Also seeing it together makes me realize that the last  `webpack` is probably redundant and unnecessary right?